system_testing/test_bucky_project/services/service_a/pc/scenarios/e2e/pc_e2e.yml

```diff
@@ -80,4 +80,52 @@ cases:
           page: index
           part:
             locate: links
-            num: 1
\ No newline at end of file
+            num: 1
+  - case_name: pc_e2e_6
+    func: pc e2e 6 get element from PageObject
+    desc: pc e2e 6 get element from PageObject
```

Comment:
Maybe 「pc e2e 6 click the single element.」is better.

Reply:
Fixed!